src/main/java/com/shopify/model/ShopifyProductUpdateRequest.java

```diff
@@ -148,57 +148,55 @@ public PublishedStep withVariantRequests(final List<ShopifyVariantRequest> varia
 				changed = true;
 			}
 
-			final List<ShopifyVariant> shopifyVariants = new ArrayList<>(variantRequests.size());
-			final List<Integer> positions = new ArrayList<>(variantRequests.size());
+			List<ShopifyVariant> shopifyVariants = new ArrayList<>(variantRequests.size());
```

Comment:
It's generally a good idea to mark something as final unless you are reassigning the object. This prevents something down the line from changing what this variable references. Can you add back in since you didn't change how these are used?